incubator/hnc/internal/validators/namespace_test.go

```diff
@@ -12,29 +12,53 @@ import (
 )
 
 func TestDeleteSubNamespace(t *testing.T) {
-	// Create a namespace with owner annotation.
+	// Create a namespace instance a.
 	sub := &corev1.Namespace{}
 	sub.Name = "a"
-	setSubAnnotation(sub, "b")
 
-	// a (subnamespace of b) -> b
-	f := foresttest.Create("B-")
-	vns := &Namespace{Forest: f}
+	// We will put an anchor a in the namespace c if it exists.
+	tests := []struct {
+		name           string
+		forest         string
+		subnamespaceOf string
+		fail           bool
+	}{
+		// There's no test case for when parent and anchor don't match, because the
```

Comment:
Do you mean: "when parent and annotation don't match"?

Reply:
No, since parent and annotation always match.  There is a mistake here in the comment in line 34, the test case is subnamespaceOf b, but I commented "subnamespaceOf c". The comment is wrong. I updated it to b in the comment.

Reply:
oh sorry somehow read it in the other way. will update..